app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivityDao.java

```diff
@@ -117,7 +94,12 @@ final void setDdbMapper(DynamoDBMapper mapper) {
             query.withExclusiveStartKey(map);
         }
         
-        QueryResultPage<DynamoScheduledActivity> queryResult = mapper.queryPage(DynamoScheduledActivity.class, query);
+        QueryResultPage<DynamoScheduledActivity> queryResult = null;
+        try {
+            queryResult = mapper.queryPage(DynamoScheduledActivity.class, query);
+        } catch(AmazonServiceException e) {
+            throw new BadRequestException(e.getErrorMessage());
+        }
```

Comment:
I agree, actually I'm wondering if we should do this in ExceptionInterceptor because there are many places where an unusual input to AWS could cause an exception... any of the 400s should probably not be returned as generic 500 errors, as ExceptionInterceptor currently does.

Right now ExceptionInterceptor looks to see if it has a BridgeServerException, and if so handles it differently. We could do the same with AmazonServiceException for the status and the error message.